lintreview/tools/py3k.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import absolute_import
+import os
+import logging
+from lintreview.tools import Tool, run_command, process_quickfix
+from lintreview.utils import in_path
+
+log = logging.getLogger(__name__)
+
+
+class Py3k(Tool):
+    """
+    $ pylint --py3k is a special mode for porting to python 3 which
+    disables other pylint checkers.
+    see https://github.com/PyCQA/pylint/issues/761
+    """
+
+    name = 'py3k'
+
+    def check_dependencies(self):
+        """
+        See if pylint is on the PATH
+        """
+        return in_path('pylint')
+
+    def match_file(self, filename):
+        base = os.path.basename(filename)
+        name, ext = os.path.splitext(base)
+        return ext == '.py'
+
+    def process_files(self, files):
+        """
+        Run code checks with pylint --py3k.
+        Only a single process is made for all files
+        to save resources.
+        """
+        log.debug('Processing %s files with %s', files, self.name)
+        command = self.make_command(files)
+        output = run_command(command, split=True, ignore_error=True)
+        if not output:
+            log.debug('No py3k errors found.')
+            return False
+
+        process_quickfix(self.problems, output, lambda name: name)
```

Comment:
You might have to filter out the goofy module headers that pylint makes..

Reply:
`process_quickfix` does ignore any lines that don't have at least two `:`s in them, but it certainly couldn't hurt.  Will do.

Reply:
Thanks :+1: